<!DOCTYPE html>
<html>
<head><title>R: RiboseQC, a Comprehensive Ribo-Seq Analysis Tool</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body><div class="container">
<h1> RiboseQC, a Comprehensive Ribo-Seq Analysis Tool
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;RiboseQC&rsquo; version 0.99.0</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../NEWS">Package NEWS</a>.</li>
</ul>

<h2>Help Pages</h2>


<table style="width: 100%;">
<tr><td style="width: 25%;"><a href="calc_cutoffs_from_profiles.html">calc_cutoffs_from_profiles</a></td>
<td>Calculate offsets from 5'end profiles</td></tr>
<tr><td style="width: 25%;"><a href="choose_readlengths.html">choose_readlengths</a></td>
<td>Filter read lengths for P-sites position calculation</td></tr>
<tr><td style="width: 25%;"><a href="create_html_report.html">create_html_report</a></td>
<td>Create the Ribo-seQC analysis report in html</td></tr>
<tr><td style="width: 25%;"><a href="create_pdfs_from_rds_objects.html">create_pdfs_from_rds_objects</a></td>
<td>Generate PDF files from RDS object files</td></tr>
<tr><td style="width: 25%;"><a href="FaFile_Circ-class.html">FaFile_Circ</a></td>
<td>A simple extension to the FaFile class that allows one to include a list of circular ranges, e.g. chrM</td></tr>
<tr><td style="width: 25%;"><a href="FaFile_Circ-class.html">FaFile_Circ-class</a></td>
<td>A simple extension to the FaFile class that allows one to include a list of circular ranges, e.g. chrM</td></tr>
<tr><td style="width: 25%;"><a href="getSeq-FaFile_Circ-method.html">getSeq-method</a></td>
<td>Yields the sequence for a particular range on a circular Fasta File note that</td></tr>
<tr><td style="width: 25%;"><a href="get_codon_usage_data.html">get_codon_usage_data</a></td>
<td>Get codon usage data (positional and bulk)</td></tr>
<tr><td style="width: 25%;"><a href="get_default_rl_selection.html">get_default_rl_selection</a></td>
<td>Get default choice of read lengths</td></tr>
<tr><td style="width: 25%;"><a href="get_metagene_data.html">get_metagene_data</a></td>
<td>Get 5'/P-site profile data for metagene analysis</td></tr>
<tr><td style="width: 25%;"><a href="get_ps_fromsplicemin.html">get_ps_fromsplicemin</a></td>
<td>Offset spliced reads on minus strand</td></tr>
<tr><td style="width: 25%;"><a href="get_ps_fromspliceplus.html">get_ps_fromspliceplus</a></td>
<td>Offset spliced reads on plus strand</td></tr>
<tr><td style="width: 25%;"><a href="get_rl_and_cutoffs.html">get_rl_and_cutoffs</a></td>
<td>Get selected read lengths and cutoffs</td></tr>
<tr><td style="width: 25%;"><a href="get_top50_all_genes.html">get_top50_all_genes</a></td>
<td>Get top 50 abundant genes (all genes)</td></tr>
<tr><td style="width: 25%;"><a href="get_top50_cds_genes.html">get_top50_cds_genes</a></td>
<td>Get top 50 abundant genes (CDS regions for protein coding genes)</td></tr>
<tr><td style="width: 25%;"><a href="get_top50_mapping.html">get_top50_mapping</a></td>
<td>Get top 50 mapping positions</td></tr>
<tr><td style="width: 25%;"><a href="load_annotation.html">load_annotation</a></td>
<td>Load genomic features and genome sequence</td></tr>
<tr><td style="width: 25%;"><a href="plot_codon_usage_bulk.html">plot_codon_usage_bulk</a></td>
<td>Plot bulk codon usage bar plots</td></tr>
<tr><td style="width: 25%;"><a href="plot_codon_usage_bulk_rmd.html">plot_codon_usage_bulk_rmd</a></td>
<td>Plot bulk codon usage bar plots within the RMarkdown document</td></tr>
<tr><td style="width: 25%;"><a href="plot_codon_usage_positional.html">plot_codon_usage_positional</a></td>
<td>Plot positional codon usage heatmap</td></tr>
<tr><td style="width: 25%;"><a href="plot_codon_usage_positional_rmd.html">plot_codon_usage_positional_rmd</a></td>
<td>Plot positional codon usage heatmaps within the RMarkdown document</td></tr>
<tr><td style="width: 25%;"><a href="plot_frame_dist_boxplot.html">plot_frame_dist_boxplot</a></td>
<td>Plot frame coverage of 5'-site profiles</td></tr>
<tr><td style="width: 25%;"><a href="plot_frame_dist_boxplot_rmd.html">plot_frame_dist_boxplot_rmd</a></td>
<td>Plot frame coverage of 5'-site profiles within the RMarkdown document</td></tr>
<tr><td style="width: 25%;"><a href="plot_metagene_bar.html">plot_metagene_bar</a></td>
<td>Plot 5'/P-site profile per read length as barplot</td></tr>
<tr><td style="width: 25%;"><a href="plot_metagene_bar_rmd.html">plot_metagene_bar_rmd</a></td>
<td>Plot 5'/P-site profile per read length as barplot in R Markdown</td></tr>
<tr><td style="width: 25%;"><a href="plot_metagene_hm.html">plot_metagene_hm</a></td>
<td>Plot 5'/P-site profiles (for all read lengths) as heatmap</td></tr>
<tr><td style="width: 25%;"><a href="plot_metagene_hm_rmd.html">plot_metagene_hm_rmd</a></td>
<td>Plot 5'/P-site profiles as heatmaps within the RMarkdown document</td></tr>
<tr><td style="width: 25%;"><a href="plot_read_biotype_dist_1.html">plot_read_biotype_dist_1</a></td>
<td>Plot read location distribution by biotype (and originating compartment)</td></tr>
<tr><td style="width: 25%;"><a href="plot_read_biotype_dist_2.html">plot_read_biotype_dist_2</a></td>
<td>Plot read location distribution by originating compartment (and biotype)</td></tr>
<tr><td style="width: 25%;"><a href="plot_read_biotype_dist_by_length.html">plot_read_biotype_dist_by_length</a></td>
<td>Plot read length and location distribution (distribution per read length)</td></tr>
<tr><td style="width: 25%;"><a href="plot_read_length_dist.html">plot_read_length_dist</a></td>
<td>Plot read length distribution</td></tr>
<tr><td style="width: 25%;"><a href="plot_read_length_dist_by_biotype.html">plot_read_length_dist_by_biotype</a></td>
<td>Plot read length and location distribution (distribution per biotype)</td></tr>
<tr><td style="width: 25%;"><a href="prepare_annotation_files.html">prepare_annotation_files</a></td>
<td>Prepare comprehensive sets of annotated genomic features</td></tr>
<tr><td style="width: 25%;"><a href="RiboseQC_analysis.html">RiboseQC_analysis</a></td>
<td>Perform a Ribo-seQC analysis</td></tr>
<tr><td style="width: 25%;"><a href="seqinfo-FaFile_Circ-method.html">seqinfo-method</a></td>
<td>Yields a seqinfo object for the FaFile_Circ with it's circular ranges slot set appropriately</td></tr>
</table>
</div></body></html>
